config/default/kustomization.yaml

```diff
@@ -11,7 +11,6 @@ namePrefix: kueue-
 labels:
   - pairs:
       app.kubernetes.io/name: kueue
-      app.kubernetes.io/component: controller
```

Comment:
Doesn't this drop the `app.kubernetes.io/component` label from the kueue-controller-manager deployment?

Reply:
We have to fix this slightly different. This specific spot adds `app.kubernetes.io/component=controller` to all the other components. I'll  update the PR.

Reply:
I pushed the fix.